tdrs-backend/setup.cfg

```diff
@@ -26,15 +26,17 @@ docstring-convention=numpy
 exclude =
     .git,
     __pycache__,
-    # We ideally want migrations to be readbale, but they often get quite nested
+    # We ideally want migrations to be readable, but they often get quite nested
     # and often have long strings.
     */migrations/*,
     # No need to check blank files
     __init__.py,
+    # These settings files often have very long strings
+    */settings/common.py/,
 # D203 conflicts with D211, which is the more conventional of the two
 extend-ignore =  E226,E302,E41,D203
 # Reducing line length so flake8 linter forces easier to read code
-max-line-length = 88
+max-line-length = 120
```

Comment:
Tweaked flake line-length per @abottoms-coder for convenience. and ignoring common.py to reduce noise